### PR TITLE
Problema de integração do banco de dados no deploy

### LINKED ISSUE #31: Problema de integração do banco de dados no deploy
## Descrição
Corrigir problema de integração do banco de dados da aplicação no deploy.

## Critérios de Aceitação

- [ ] O deploy da aplicação deve está funcionando.
- [ ] O banco de dados da aplicação deve está funcionando no deploy.
- [ ] Todas as operações com o banco de dados deve está funcional no deploy.

### PR DESCRIPTION
**Pull request by lucianosz7**
# Problema de integração do banco de dados no deploy

## Descrição
Tornar a API acessível localmente em ambiente de desenvolvimento. Para que o app possa ser usado no celular para testes de funcionamento

[Problema de integração do banco de dados no deploy](#31 )

## Porque este Pull Request é necessário?
Para que seja possível utilizar todas as funcionalidade da aplicação que necessitem do banco de dados.

## Critérios de Aceitação
- [X] O deploy da aplicação deve está funcionando.
- [X] O banco de dados da aplicação deve está funcionando no deploy.
- [X] Todas as operações com o banco de dados deve está funcional no deploy.

Resolve #31

### CODE REVIEW

File: api/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from .settings import *
```
Comment (lucianosz7) [inline api/__init__.py]: Codacy Issue found: ['.settings.*' imported but unused](https://app.codacy.com/app/ArBC/2019.2-ArBC-API/pullRequest?prid=4554355)

**Comment by lucianosz7** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- api/env.py  2
- app/forms.py  1
         

Clones added
============
- api/dev.py  3
- api/settings.py  2
- api/prod.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/2019.2-ArBC-API/2019.2-ArBC-API/pullRequest?prid=4554355&bid=15318744)

File: api/__init__.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+from .env import credentials
```
Comment (lucianosz7) [inline api/__init__.py]: Codacy Issue found: ['.env.credentials' imported but unused](https://app.codacy.com/app/ArBC/2019.2-ArBC-API/pullRequest?prid=4554355)

**Comment by lucianosz7** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- app/forms.py  1
- api/env.py  2
         

Clones added
============
- api/settings.py  2
- api/dev.py  3
- api/prod.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/2019.2-ArBC-API/2019.2-ArBC-API/pullRequest?prid=4554355&bid=15318744)

File: api/__init__.py (hunk 1 above)
Comment (lucianosz7) [inline api/__init__.py]: Codacy Issue found: ['.settings.*' imported but unused](https://app.codacy.com/app/ArBC/2019.2-ArBC-API/pullRequest?prid=4554355)

File: api/__init__.py (hunk 2 above)
Comment (lucianosz7) [inline api/__init__.py]: Codacy Issue found: ['.env.credentials' imported but unused](https://app.codacy.com/app/ArBC/2019.2-ArBC-API/pullRequest?prid=4554355)

**Comment by lucianosz7** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- api/env.py  2
- app/forms.py  1
         

Clones added
============
- api/settings.py  2
- api/dev.py  3
- api/prod.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/2019.2-ArBC-API/2019.2-ArBC-API/pullRequest?prid=4554355&bid=15318744)

File: api/__init__.py (hunk 1 above)
Comment (lucianosz7) [inline api/__init__.py]: Codacy Issue found: ['.settings.*' imported but unused](https://app.codacy.com/app/ArBC/2019.2-ArBC-API/pullRequest?prid=4554355)

File: api/__init__.py (hunk 2 above)
Comment (lucianosz7) [inline api/__init__.py]: Codacy Issue found: ['.env.credentials' imported but unused](https://app.codacy.com/app/ArBC/2019.2-ArBC-API/pullRequest?prid=4554355)

**Comment by lucianosz7** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- app/forms.py  1
- api/env.py  2
         

Clones added
============
- api/dev.py  3
- api/settings.py  2
- api/prod.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/2019.2-ArBC-API/2019.2-ArBC-API/pullRequest?prid=4554355&bid=15318744)

**Comment by lucianosz7** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- app/forms.py  1
- api/env.py  2
         

Clones added
============
- api/prod.py  3
- api/settings.py  2
- api/dev.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/2019.2-ArBC-API/2019.2-ArBC-API/pullRequest?prid=4554355&bid=15318744)

**Review by igor-paiva — APPROVED** [review]
:+1:

**Comment by eilimrib** [comment]
Ceritinho.

**Pull request merged**